bundlesize.config.json

```diff
@@ -78,11 +78,11 @@
     },
     {
       "path": "static/build/page-admin.css",
-      "maxSize": "25KB"
+      "maxSize": "26KB"
```

Comment:
I've surprised that this size needed to be adjusted.  I wonder if our Less references need investigation.